v19.2/drop-table.md

```diff
@@ -62,52 +65,82 @@ DROP TABLE
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW TABLES FROM bank;
+> SHOW TABLES FROM movr;
 ~~~
 
 ~~~
-+--------------------+
-| table_name         |
-+--------------------+
-| accounts           |
-| user_accounts_view |
-+--------------------+
-(2 rows)
+          table_name
++----------------------------+
+  rides
+  user_promo_codes
+  users
+  vehicle_location_histories
+  vehicles
+(5 rows)
 ~~~
 
 ### Remove a table and dependent objects with `CASCADE`
 
-In this example, a view depends on the table being dropped. Therefore, it's only possible to drop the table while simultaneously dropping the dependent view using `CASCADE`.
+In this example, a foreign key from a different table references the table being dropped. Therefore, it's only possible to drop the table while simultaneously dropping the dependent foreign key constraint using `CASCADE`.
 
 {{site.data.alerts.callout_danger}}<code>CASCADE</code> drops <em>all</em> dependent objects without listing them, which can lead to inadvertent and difficult-to-recover losses. To avoid potential harm, we recommend dropping objects individually in most cases.{{site.data.alerts.end}}
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW TABLES FROM bank;
+> SHOW TABLES FROM movr;
+~~~
+
+~~~
+          table_name
++----------------------------+
+  rides
+  user_promo_codes
+  users
+  vehicle_location_histories
+  vehicles
+(5 rows)
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> DROP TABLE users;
 ~~~
 
 ~~~
-+--------------------+
-| table_name         |
-+--------------------+
-| accounts           |
-| user_accounts_view |
-+--------------------+
-(2 rows)
+pq: "users" is referenced by foreign key from table "vehicles"
 ~~~
 
+You can use the [`SHOW CREATE`](show-create.html) statement to see how the columns in a table are created, including data types, default values, indexes, and constraints.
```

Comment:
It might be nice to preface this with some context, something like "To see how 'users' is referenced from 'vehicles', use the `SHOW CREATE` ..."

Reply:
Done.